src/components/Footer/FooterPrimary/StyledFooterPrimary.js

```diff
@@ -1,18 +1,78 @@
 import styled from 'styled-components'
 import { resolveMedia } from '../../../utils/helpers'
 import { Row, Col } from '../../Grid'
+import { Anchor } from '../../Anchor'
+import { Paragraph } from '../../Paragraph'
 
+// row container
 const FooterPrimaryRow = styled(Row)`
   ${props =>
     !props.background && `background-color: ${props.theme.colors.alternate};`}
-  ${props => !props.padding && `padding: 24px 0 12px;`}
+  ${props => !props.padding && `padding: 24px 15px 12px;`}
 `
 
-const SocialMediaLinks = styled(Col)``
+// social media section
+const SocialContainer = styled(Col)`
+  ${props => !props.padding && `padding: 12px 0;`}
+`
+
+const SocialLink = styled(Col)`
+  flex-grow: 0;
+`
+
+const SocialLogo = styled(Anchor)`
+  ${props => props.facebook && `background-color: #3c5a99;`}
+  ${props => props.twitter && `background-color: #2ba9e1;`}
+  ${props => props.youtube && `background-color: #cd201f;`}
+
+  border-radius: 50%;
+  width: 60px; 
+  height: 60px;
+  line-height: 57px;
```

Comment:
Pretty trivial I know but maybe `line-height` should be the same value as the height.